.github/config/localhost.yaml

```diff
@@ -6,6 +6,7 @@ transport: core.local
 scheduler: core.direct
 shebang: '#!/usr/bin/env bash'
 work_dir: PLACEHOLDER_WORK_DIR
+default_memory_per_machine: 0 # 0 value means "not specified".
```

Comment:
Still think we should remove this. It should be perfectly fine to not set anything.

Reply:
This particular file is used to set `localhost` in interactive mode. So removing this would result in aiida requesting the value:
```
Default amount of memory per machine (kB).:
```

Setting `required=False` as you suggest further for `DEFAULT_MEMORY_PER_MACHINE ` doesn't change anything in this respect.

So I still don't understand how can this be removed.

Reply:
When you say that "this file is used to set `localhost` in interactive mode" do you mean the tests of the GitHub Actions workflow? In that case, the setup _should_ simply use non-interactive mode. That is what the mode is for: it should be used when in principle all options are defined and the command shouldn't prompt.

Reply:
> When you say that "this file is used to set `localhost` in interactive mode" do you mean the tests of the GitHub Actions workflow?

yes



> In that case, the setup _should_ simply use non-interactive mode. That is what the mode is for: it should be used when in principle all options are defined and the command shouldn't prompt.

ok, done.